flink-rpc/flink-rpc-akka/src/test/java/org/apache/flink/runtime/rpc/akka/AkkaRpcActorTest.java

```diff
@@ -299,7 +299,7 @@ void testActorTerminationWhenServiceShutdown() throws Exception {
 
             CompletableFuture<Void> terminationFuture = rpcEndpoint.getTerminationFuture();
 
-            rpcService.stopService();
+            rpcService.closeAsync();
```

Comment:
A bit out-of-scope, but I'm wondering whether we should also assert the completion of the `CompletableFuture` that is returned by `closeAsync` here? ...just to be sure that the service has actually stopped properly after the completion of the `terminationFuture` succeeded. WDYT? :thinking:

Reply:
AFAICT that's covered by `AkkaRpcServiceTest#testAkkaRpcServiceShutDownWithRpcEndpoints`